### PR TITLE
[Update] Sample categories - first batch

### PR DESCRIPTION
**Pull request by CalebRas**
## Description

This PR updates the "category" in the metadata file to match the [https://github.com/Esri/arcgis-maps-sdk-kotlin-samples](Kotlin samples) and the [common design sample categories](https://devtopia.esri.com/runtime/common-samples/blob/Ting/Update-category-design/designs/categories.md) for the first half of the swift samples. 

## Linked Issue(s)

- `swift/issues/4227`
- Common Design PR: [#3856](https://devtopia.esri.com/runtime/common-samples/pull/3856)

## How To Test

* Verify each sample category matches the corresponding Kotlin sample and/or the correct common design sample category.

### CODE REVIEW

**Review by yo1995 — COMMENTED** [review]
Thanks for making these changes! Please let me know when it is ready for review.

**Comment by CalebRas** [comment]
@yo1995 This first batch should be good to review now since the [common design PR](https://devtopia.esri.com/runtime/common-samples/pull/3856) was merged.

File: Shared/Samples/Monitor changes to map load status/README.metadata.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
-    "category": "Maps",
+    "category": "Maps ",
```
Comment (yo1995) [inline Shared/Samples/Monitor changes to map load status/README.metadata.json:2]:
```suggestion
    "category": "Maps",
```

**Review by yo1995 — APPROVED** [review]

**Review by yo1995 — APPROVED** [review]
Thank you. Since these changes are not related to the API, I'll skip the review by someone else and merge this PR directly.

**Pull request merged**